solutions/security/get-started/automatic-migration.md

```diff
@@ -0,0 +1,156 @@
+3. Next, under **Migrate rules & add data**, click **Translate your existing SIEM rules to Elastic**, then **Upload rules**.
+4. Follow the instructions on the **Upload Splunk SIEM rules** flyout to export your rules from Splunk as JSON. 
+
+   :::{image} ../../../images/security-siem-migration-1.png
+   :alt: the Upload Splunk SIEM rules flyout
+   :width: 700px
+   :screenshot:
+   :::
+
+
+   ::::{note}
+   The provided query downloads Splunk correlation rules and saved searches. Alternatively, as long as you    export your results in a JSON format, you can use a different query. For example:
+
+   ```
+   | rest /servicesNS/-/-/saved/searches
+   | search is_scheduled=1 AND eai:acl.app=splunksysmonsecurity
+   | where disabled=0
+   | table id, title, search, description, action.escu.eli5, 
+   ```
+   Which would download rules related to just the `splunksysmonsecurity` app.
+
+   We don't recommend downloading all searches (for example with `| rest /servicesNS/-/-/saved/searches`) since most of the data will be irrelevant to SIEM rule migration. 
+   ::::
+
+5. Select your JSON file and click **Upload**. 
+   ::::{note}
+   If the file is large, you may need to separate it into multiple parts and upload them individually to avoid exceeding your LLM's context window.
+   ::::
+
+6. After you upload your Splunk rules, Automatic Migration will detect whether they use any Splunk macros or lookups. If so, follow the instructions which appear to export and upload them. Alternatively, you can complete this step later — however, until you upload them, some of your migrated rules will have a `partially translated` status. If you upload them now, you don't have to wait on the page for them to be processed — a notification will appear when processing is complete.
+
+7. Click **Translate** to start the rule translation process. You don't need to stay on this page. A notification will appear when the process is complete. 
+
+8. When migration is complete, click the notification or return to the **Get started** page then click **View translated rules** to open the **Translated rules** page. 
+
+
+## The Translated rules page
+
+This section describes the **Translated rules** page's interface and explains how the data that appears here is derived. 
+
+When you upload a new batch of rules, they are assigned a name and number, for example `SIEM rule migration 1`, or `SIEM rule migration 2`. Use the **Migrations** dropdown menu in the upper right to select which batch appears. 
+
+::::{image} ../../../images/security-siem-migration-processed-rules.png
+:alt: The translated rules page
+:width: 850px
+:screenshot:
+::::
+
+The table's fields are as follows:
+
+* **Name:** The names of Elastic authored rules cannot be edited until after rule installation. To edit the name of a custom translated rule, click the name and select **Edit**.
```

Comment:
There's some variation in the way this is constructed throughout the page. Our style guide recommends hyphenating words that create a compound adjective, so that's what I would suggest: https://docs.elastic.dev/tech-writing-guidelines/grammar-spelling#dashes-and-hyphens

```suggestion
* **Name:** The names of Elastic-authored rules cannot be edited until after rule installation. To edit the name of a custom translated rule, click the name and select **Edit**.
```